Initial Demo

msr-lovely opened the pull request.

msr-lovely committed:
demo

msr-lovely committed:
Update README.md

msr-lovely committed:
Update index.js

msr-lovely committed:
Update README.md

jprester commented:
Can you use the color scheme as in the design?

For example, the blue button color should be #1D7EA2. The header background color should be: #000000 etc..

It would be also nice to see how you update the theme styles cause I am trying to figure it out.

jprester commented on package.json:
```diff
@@ -1,39 +1,20 @@
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "dev": "next dev",
```
this is a minor thing but a lot of projects that people use to start with the default port of 3000, and in that case, the app won't run if something is already running there. Maybe you can just write it to sue different port like 3005

"dev": "next dev -p 3005"

msr-lovely replied:
This isn't going to be SSR (server side rendering) so no need for server port at all. dev is development, which is testing. When published on Netlify, there's no need to use ports.

The build command is npm run build, so npm run dev is actually not used on production :3

jprester commented:
And if possible can you add a header with logo like it is in design? Mostly so I can learn how to use this template approach.

jprester commented on pages/summaries/pick.js:
```diff
@@ -0,0 +1,76 @@
+        onChange: setChosenSummary
+    })
+    const group = getRootProps()
+    const confirmChoise = () => {
```
change spelling confirmChoise => confirmChoice

msr-lovely committed:
navbar, fixed typo, theme examples, theme adjustment

theme settings at style/theme.js

msr-lovely commented:
> 
> 
> And if possible can you add a header with logo like it is in design? Mostly so I can learn how to use this template approach.

I added navigation bar with logo and title~

> 
> 
> Can you use the color scheme as in the design?
> 
> For example, the blue button color should be #1D7EA2. The header background color should be: #0 etc..
> 
> It would be also nice to see how you update the theme styles cause I am trying to figure it out.

So I changed the blue color to match the current theme, so the blue is #0bf

msr-lovely committed:
Update Layout.js
